protocols/mempool-overview.md

```diff
@@ -105,25 +83,30 @@ To this end, we first provide a (brief) definition of what is a ledger, that is
 At a process $p$, we shall write $p.var$ the local variable $var$ at $p$.
 
 **Ledger.**
-We use the standard defintion of (BFT) SMR, where each process $p$ has a ledger, written $p.ledger$.
+We use the standard definition of (BFT) SMR in the context of blockchain, where each process $p$ has a ledger, written $p.ledger$.
 At process $p$, the $i$-th entry of the ledger is denoted $p.ledger[i]$.
 This entry contains either a null value ($\bot$), or a set of transactions, aka., a block.
-The height of the ledger at $p$ is the index of the first null entry.
+The height of the ledger at $p$ is the index of the first null entry; denoted $p.height$.
 Operation $submit(txs, i)$ attempts to write the set of transactions $txs$ to the $i$-th entry of the ledger.
-As standard, the ledger ensures that there is no gap between two entries at each process,
-that is  $\forall i. \forall p. p.ledger[i] \neq \bot \implies (i=0 \vee p.ledger[i-1] \neq \bot)$.
-It also makes sure that no two correct processes have different ledger entries (agreement);
-formally: $\forall i. \forall p,q \in Correct. (p.ledger[i] = \bot) \vee (q.ledger[i] = \bot) \vee (p.ledger[i] = q.ledger[i])$.
-Finally, the ledger requires that if some transaction appears at an index $i$, then a process submitted it at that index (validity).
-All the transactions in the non-null entries of the ledger are denoted $p.committed$;
-formally $p.committed = \\{ tx : \exists j. tx \in p.ledger[j] \\}$.
-The (history) variable $p.submitted$ holds all the transactions submitted so far by $p$.
+The (history) variable $p.submitted[i]$ holds all the transactions (if any) submitted by $p$ at height $i$.
+By extension, $p.submitted$ are all the transaction submitted by $p$.
+A transaction is committed when it appears in one of the entries of the ledger.
+We write $p.committed$ the committed transactions at $p$.
+
+As standard, the ledger ensures that:  
+* _(Gap-freedom)_ There is no gap between two entries at a correct process:  
+$\forall i \in 	\mathbb{N}. \forall p \in Correct. \square(p.ledger[i] \neq \bot \implies (i=0 \vee p.ledger[i-1] \neq \bot))$;  
+* _(Agreement)_ No two correct processes have different ledger entries; formally:  
+$\forall i \in 	\mathbb{N}. \forall p,q \in Correct. \square((p.ledger[i] = \bot) \vee (q.ledger[i] = \bot) \vee (p.ledger[i] = q.ledger[i]))$;  
+* _(Validity)_ If some transaction appears at an index $i$ at a correct process, then a process submitted it at that index:  
+$\forall p \in Correct. \exists q \in Processes. \forall i \in 	\mathbb{N}. \square(tx \in p.ledger[i] \implies tx \in \bigcup_q q.submitted[i]$).
+* _(Termination)_ If a correct process submits a block at its current height, eventually its height get incremented:  
```

Comment:
The definition does not really uses the definition of block up to here. Processes submit transactions.

Reply:
What it looks to be the spirit of this property is that the height of the blockchain increases, meaning that new blocks of transactions are appended to it, if any correct process submits new transactions to append to the blockchain.